public/app/features/admin/UserCreatePage.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useCallback } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { Forms } from '@grafana/ui';
+import { NavModel } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { StoreState } from '../../types';
+import { getNavModel } from '../../core/selectors/navModel';
+import Page from 'app/core/components/Page/Page';
+import { updateLocation } from 'app/core/actions';
+
+interface UserCreatePageProps {
+  navModel: NavModel;
+  updateLocation: typeof updateLocation;
+}
+
+interface UserDTO {
+  name: string;
+  password: string;
+  email?: string;
+  login?: string;
+}
+
+const createUser = async (user: UserDTO) => getBackendSrv().post('/api/admin/users', user);
```

Comment:
If this request fails, how do we handle the errors? Do we have any globally defined error handling?

Reply:
Yes, an alert will be shown